test/techHttpSpec.js

```diff
@@ -37,7 +57,7 @@ describe("tech-http", function() {
 
         return http.get("avop-ruuid-42", "http://foo:8080/api/bar/baz").then(function(result) {
             assert.ok(result.foo === "bar", "request apis should have been wrapped");
-            assert.ok(parseFloat(duration.ms) > 100, "the request time should have been logged");
+            assert.ok(parseFloat(duration.ms) > 90, "the request time should have been logged");
```

Comment:
OK, reading the full file, I understand that it's because of the `setTimeout(..., 100)`